dvl-a50/mavlink2resthelper.py

```diff
@@ -245,7 +294,20 @@ def set_param(self, param_name, param_type, param_value):
             result = requests.post(MAVLINK2REST_URL + '/mavlink', json=data)
             return result.status_code == 200
         except Exception as error:
-            print("Error setting parameter: " + str(error))
+            logger.warning("Error setting parameter: " + str(error))
```

Comment:
```suggestion
            logger.warning(f"Error setting parameter: {error}")
```

Reply:
good, I also added the parameter failed to set